$MirOS: ports/print/cups/patches/patch-config-scripts_cups-libtool_m4,v 1.1 2005/12/16 01:08:10 bsiegert Exp $
--- config-scripts/cups-libtool.m4.orig	Wed Apr 26 19:52:04 2006
+++ config-scripts/cups-libtool.m4	Mon Jun 19 19:41:10 2006
@@ -22,27 +22,14 @@ dnl       EMail: cups-info@cups.org
 dnl         WWW: http://www.cups.org
 dnl
 
-AC_ARG_ENABLE(libtool_unsupported, [  --enable-libtool-unsupported=LIBTOOL_PATH
-                          turn on building with libtool (UNSUPPORTED!), default=no],
-	[if test x$enable_libtool_unsupported != xno; then
-		LIBTOOL="$enable_libtool_unsupported"
-		enable_shared=no
-		echo "WARNING: libtool is not supported or endorsed by Easy Software Products."
-		echo "         WE DO NOT PROVIDE TECHNICAL SUPPORT FOR LIBTOOL PROBLEMS."
-		echo "         (even if you have a support contract)"
-	else
-		LIBTOOL=""
-	fi])
+AC_PROG_LIBTOOL
 
-AC_SUBST(LIBTOOL)
-
-if test x$LIBTOOL != x; then
 	LIBCUPS="libcups.la"
 	LIBCUPSIMAGE="libcupsimage.la"
 	LINKCUPS="../cups/\$(LIBCUPS)"
 	LINKCUPSIMAGE="../filter/\$(LIBCUPSIMAGE)"
-	DSO="\$(CC)"
-fi
+	DSO="\$(LIBTOOL) --mode=link \$(CC)"
+	DSOFLAGS="$DSOFLAGS -rpath \$(libdir) \$(OPTIM)"
 
 dnl
 dnl End of "$Id$".
